src/services/membership_api/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const mongoose = require('mongoose');
+
+const app = express();
+
+const helloMembers = require('./routes/memberRoute');
+app.use(helloMembers);
+
+app.listen(3000, () => console.log('connected to server'));
```

Comment:
Save the PORT and other environment variables in the .env and add it to the .gitignore

Reply:
changes done

Reply:
done